### ISSUE TITLE
Studio: multiple domains possibly with mixed-types

### ISSUE THREAD

**Issue by sheryjoe** [SCIInstitute/ShapeWorks#579]
- Studio should handle multiple domains per shape given the new spreadsheet project file. 
- Domains could be of mixed-types, e.g. LA mesh (or segmentation) and LAA ostium (contour).
- Optimization should be in the joint shape space
- Modes and mean should be visualized without ad-hoc transformations.

**Comment by akenmorris** [comment]
What are "ad-hoc transformations"? @sheryjoe

**Comment by sheryjoe** [comment]
> What are "ad-hoc transformations"? @sheryjoe

I think that was related to a discussion where you mentioned that you had to translate one domain (in ShapeWorksView2) to see both domains simultaneously.

**Comment by akenmorris** [comment]
Right, when the geometry is overlapping due to grooming and/or correspondence extrapolation.  I'm not sure how to solve the general problem.

**Comment by iyerkrithika21** [comment]
@akenmorris  any thoughts/updates on this?
For #683

**Comment by akenmorris** [comment]
I don't have any updates right now.  It will be a fair amount of work.

**Comment by sheryjoe** [comment]
@medakk related to shared boundaries.
